elements/pfe-autocomplete/src/pfe-search-droplist.scss

```diff
@@ -8,7 +8,8 @@
   line-height: pfe-var(line-height);
 }
 
-:host([open]) {
+:host([open]),
+:host([pfe-c-open]) {
```

Comment:
Since open is alias'ed, I'm not sure we need both in the CSS.  Probably can just use `pfe-c-open`.

Reply:
Agree :+1: